song-go-client/cmd/search.go

```diff
@@ -21,9 +21,10 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
+	"strings"
+
```

Comment:
this space is just to distinguish from the urls, or?

Reply:
correct. but it's prettier adding the space.